x-pack/plugins/watcher/public/models/watch/threshold_watch.js

```diff
@@ -91,28 +99,102 @@ export class ThresholdWatch extends BaseWatch {
       index: [],
       timeField: [],
       triggerIntervalSize: [],
+      aggField: [],
+      termSize: [],
+      termField: [],
+      threshold: [],
+      timeWindowSize: [],
     };
     validationResult.errors = errors;
     if (!this.name) {
-      errors.name.push(i18n.translate('xpack.watcher.sections.watchEdit.threshold.error.requiredNameText', {
-        defaultMessage: 'Name is required',
-      }));
+      errors.name.push(
+        i18n.translate('xpack.watcher.sections.watchEdit.threshold.error.requiredNameText', {
+          defaultMessage: 'Name is required',
+        })
+      );
     }
     if (this.index !== undefined && this.index.length < 1) {
-      errors.index.push(i18n.translate('xpack.watcher.sections.watchEdit.titlePanel.enterOneOrMoreIndicesValidationMessage', {
-        defaultMessage: 'Enter one or more indices'
-      }));
+      errors.index.push(
+        i18n.translate(
+          'xpack.watcher.sections.watchEdit.titlePanel.enterOneOrMoreIndicesValidationMessage',
```

Comment:
nit: should the `i18n` strings follow a consistent pattern?

`xpack.watcher.sections.watchEdit.threshold.error.requiredNameText` (above) vs.
`xpack.watcher.sections.watchEdit.titlePanel.enterOneOrMoreIndicesValidationMessage`

Reply:
I think we'll want to go through and fix all these at some point.  These were copied from the existing ones, but I think I'd rather fix the naming all in one go as a different PR.